lib/stat_tracker.rb

```diff
@@ -15,6 +18,21 @@ def initialize(game_path, team_path, game_teams_path)
     @team_path = team_path
     @game_teams_path = game_teams_path
     # @team_collection = TeamCollection.new(team_path)
-    # @game_collection = GameCollection.new(game_path, game_teams_path)
+    @game_collection = GameCollection.new(game_path)
+    @games = @game_collection.all_games
+  end
+
+# Game Statistics Tests - Helper Methods #
+  def total_goals_per_game
+    result = {}
+    @games.each do |game|
+      result[game.game_id] = game.away_goals + game.home_goals
+    end
```

Comment:
You could refactor this using reduce! Looks like the typical use case for it!

Reply:
@guardiac just pushed these changes! I was struggling with reduce the first time because the keys (game_id) were integers. I didn't realize that integers cannot be keys and also that symbols can't be created from integers 🤯. Because of this, I converted the game_ids to strings and updated the tests accordingly.

Reply:
I thought the integers as keys part was strange just because I had never seen it before. But it worked for me when I was reviewing your original changes. This is what I tried before suggesting the change:

```
def total_goals_per_game
    @games.reduce({}) do |result, game|
      result[game.game_id] = game.away_goals + game.home_goals
      result
    end
  end
```
That didn't work for you?


 Also had no idea you can't make symbols from integers!

Reply:
@guardiac ahhh you are totally right! I must have been doing something funky in my test or something because integer as key does work now. I updated to reflect this.